addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -34,10 +34,6 @@
     'uom.product_uom_gal': 'GLL',
     'uom.product_uom_cubic_inch': 'INQ',
     'uom.product_uom_cubic_foot': 'FTQ',
-    'uom.uom_square_meter': 'MTK',
```

Comment:
I would just not remove these here. So we stay consistent between the 2 dicts. (they should be the same tbh, but I suppose the ubl one existed before)

In master I would remove this logic to depend on uom too (which would even remove the need to override in l10n_tr_nilvera_invoice).